store/store.go

```diff
@@ -90,6 +91,7 @@ func (a *Alerts) gc() {
 	}
 	a.Unlock()
 	a.cb(resolved)
```

Comment:
@gotjosh I want to remove the callback in a future PR, so I'm not too worried about both returning `resolved` and passing it to the callback.